tetramer_validator/data/molecule.tsv

```diff
@@ -0,0 +1,695 @@
+Label	IEDB Label	Synonyms	Restriction Level

```

Comment:
I don't think that the Restriction Level column is needed. Please remove it.

Reply:
https://github.com/IEDB/tetramer-validator/pull/17/commits/d4c8f2b1cb2d2bb7c78b64648c5c778e1426c84f